python_anvil/api.py

```diff
@@ -117,6 +117,10 @@ def request_rest(self, options: Optional[dict] = None):
         api = RestRequest(self.client, options=options)
         return api
 
+    def request_fully_qualified(self, options: Optional[dict] = None):
```

Comment:
idt this is actually used anywhere - looks like the func above is used [here](https://github.com/anvilco/python-anvil/blob/28e4d3f8ddf73e1f14979debef857a717979792d/python_anvil/api.py#L156). a couple other placess too, probably should use this new version

Reply:
Yeah, request_rest also doesnt seem to be called anywhere. I guess someone could use these if they wanted to when making other requests to Anvil